deployment/docker/README.md

```diff
@@ -154,6 +154,7 @@ Variable name | Meaning | Possible values | Default value |
 ` DRAFT_ENABLED`|Enable draft feature|true/false
 `DRAFT_POLLING_RATE`|Control draft timing||15000
 `EXPORT_PDF_ENABLED`|Manage export to pdf feature|true/false
+`PUBLIC_WORKFLOW_ENABLED`|Create public or private workflow|true/false
```

Comment:
please change description to "Enable creating workflow for all tenants"

Reply:
done